notebook/services/kernels/kernelmanager.py

```diff
@@ -130,6 +136,11 @@ def __init__(self, **kwargs):
     # Methods for managing kernels and sessions
     #-------------------------------------------------------------------------
 
+    def __init__(self, **kwargs):
+        self.super = MultiKernelManager
```

Comment:
Clever! I was slightly uncomfortable with this when I first saw it—I thought you just wanted an alias for `super(...)`, but I now recognize this is the magic for the class hierarchy. 

That said, I have two recommendations... 

1. I think we need to add some thorough inline comments here (and below, in `AsyncMultiKernelManager`. Especially using the attr name `super`, I'm afraid someone reading this a few years from now might think this is a mistake... that you actually mean super().
2. Maybe even consider renaming this attribute? Something like `base_multikernel_manager` or something that makes it clear you're calling a class other than super().

Reply:
I agree it could be confusing to name it `super`, my bad!